eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -77,15 +79,34 @@
     @SuppressWarnings("PMD.ImmutableField")
     private Objectionary objectionary = new RemoteObjectionary();
 
+    /**
+     * Target directory.
+     * @checkstyle MemberNameCheck (7 lines)
+     */
+    @SuppressWarnings("PMD.ImmutableField")
+    private Path outputPath = Paths.get(
```

Comment:
@marceloamadeu 
You can use `Path#resovle` to avoid having `file.separator` here.

Reply:
@andreoss It's done...please check.
tks

Reply:
@andreoss It's done...please check.
tks